pkgs/applications/misc/nb/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, lib, fetchFromGitHub, installShellFiles
+, bash, git
+}:
+stdenv.mkDerivation rec {
+  version = "5.7.8";
+  pname = "nb";
+
+  src = fetchFromGitHub {
+    owner = "xwmx";
+    repo = "nb";
+    rev = version;
+    sha256 = "1sw09sgskzsn3dy4ipvl52hwh5k3cwqfhjnm7rmxbzcpfk3jm3sp";
+  };
+
+  buildInputs = [ bash git ];
```

Comment:
How is git used here? If it is used to extract version or commit information from the repository this would require patching.

Reply:
The script uses git to synchronize your notes. So at runtime, not buildtime.

Reply:
Shouldn't makeWrapper be used for such things?